src/config/data.js

```diff
@@ -160,10 +138,77 @@ const upcomingBootCamps = [
     image: LONDON_LOCATION,
     mapLink: 'https://goo.gl/maps/jjX9zs5Ags32',
   },
+  {
+    dates: '12-18 May, 2019',
+    dateStartsOn: new Date('2019-05-12T18:00:00'),
+    city: LONDON,
+    country: 'UK',
+    type: REACT_BOOTCAMP,
+    cityShortName: 'London',
+    location: 'Publicis.Sapient - Eden House, 8 Spital Square',
+    pathUrl: '/react-redux-graphql-bootcamp-london/',
+    trainingInstanceId: '5cacbaec80c1577e379810df',
+    price: 1800,
+    currency: 'gbp',
+    image: LONDON_LOCATION,
+    mapLink: 'https://goo.gl/maps/jjX9zs5Ags32',
+  },
+]
+
+const upcomingAdvanced = [
+  {
+    dates: '15-18 May, 2019',
+    dateStartsOn: new Date('2019-05-15T09:00:00'),
+    city: LONDON,
+    country: 'UK',
+    type: ADVANCED_REACT,
+    cityShortName: 'London',
+    location: 'Publicis.Sapient - Eden House, 8 Spital Square',
+    pathUrl: '/advanced-react-redux-graphql-bootcamp-london',
+    trainingInstanceId: '5cacbe8580c1577e379810e0',
+    price: 1250,
+    currency: 'gbp',
+    image: LONDON_LOCATION,
+  },
+  {
+    dates: '10-13 July, 2019',
+    dateStartsOn: new Date('2019-07-10T09:00:00'),
+    city: LONDON,
+    country: 'UK',
+    type: ADVANCED_REACT,
+    cityShortName: 'London',
+    location: 'Publicis.Sapient - Eden House, 8 Spital Square',
+    pathUrl: '/advanced-react-redux-graphql-bootcamp-london/2',
+    trainingInstanceId: '5cacbfae80c1577e379810e1',
+    price: 1250,
+    currency: 'gbp',
+    image: LONDON_LOCATION,
+  },
 ]
 export const trainings = [
   ...upcomingPartTimes,
   ...upcomingBootCamps,
+  ...upcomingAdvanced,
+  {
+    dates: 'March 31 to April 6, 2019',
```

Comment:
@ric9176 are we still using this one? should we remove the data that is in the past? it's a bit messy otherwise

Reply:
true, I'll delete all past ones and push